onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selector_action_transformer.cc

```diff
@@ -228,6 +228,28 @@
 #endif
 }
 
+void DQMatMulQDQRules(SelectorActionRegistry& qdq_selector_action_registry) {
+  // 2 nodes. DQ -> MatMul. DQ is the second input to MatMul.
+  // DQ's weight is int4/uint4. DQ's scale is float/float16.
+  // DQ is block-quantized along axis 0, with block_size >= 16 and as 2's power.
+  const std::string action_name{"DQMatMul"};
+
+  std::unique_ptr<Action> action = std::make_unique<QDQ::DQMatMulReplaceWithMatMulNBits>();
```

Comment:
>DQMatMulReplaceWithMatMulNBits [](http://example.com/codeflow?start=57&length=30)

@yufenglee how to pass in accuracy_level?

Reply:
as we discussed yesterday, we may need to add a session option to contrl the accuracy_level.

Reply:
added accuracy level to session options